package.json

```diff
@@ -55,7 +55,8 @@
         "embla-carousel-react": "^7.1.0",
         "is-absolute-url": "^4.0.1",
         "jsonpath": "^1.1.1",
-        "recharts": "^2.12.0"
+        "recharts": "^2.12.0",
+	"embla-carousel-autoplay": "^7.1.0"
```

Comment:
nit: let's insert this alphabetically. But also, shouldn't there be a corresponding `package-lock.json` update?